test/public/logs/overview.test.js

```diff
@@ -189,8 +189,8 @@ module.exports = () => {
 
         // Insert some text into the filter
         await page.type('#authorFilterText', 'Jane');
-        // Wait until there is only 1 row after filtering
-        await page.waitForFunction(() => document.querySelectorAll('table tbody tr').length === 1, { timeout: 1500 });
+        // Wait until there are no row after filtering
```

Comment:
You can remove this kind of comment, code is quite explicit now

Reply:
Done